packages/block-library/src/button/edit.js

```diff
@@ -72,9 +85,134 @@ function BorderPanel( { borderRadius = '', setAttributes } ) {
 	);
 }
 
+function ToolbarMovers( { clientId } ) {
```

Comment:
Instead of these custom movers, I think we should just use the vertical movers used by the navigation block.

Reply:
Right, this was added as an experiment to compare how movers in the toolbar would look like. Now inner blocks already supports movers in the toolbar so I removed this code and relied on InnerBlocks.